File: SoftUni-Python-Fundamentals/p04_functions/exercises/odd_and_even_sum.py
def sum_even_odd(numbers):
    odd_sum = 0
    even_sum = 0
    for num in range(0, len(numbers)):
        if int(numbers[num]) % 2 == 0:
            even_sum += int(numbers[num])
        elif int(numbers[num]) % 2 == 1:
            odd_sum += int(numbers[num])
    return f"Odd sum = {odd_sum}, Even sum = {even_sum}"


number = str(input())
print(sum_even_odd(number))

File: SoftUni-JS-Front-End/functionsAndStatementsExercise/oddAndEvenSum.js
function solve(number) {
    let numbers = number.toString().split("")
    let evenSum = 0
    let oddSum = 0
    for (let i of numbers) {
        if (i % 2 === 0) {
            evenSum += Number(i)
        }else {
            oddSum += Number(i)
        }
    }
    return `Odd sum = ${oddSum}, Even sum = ${evenSum}`
}

console.log(solve(1000435))